### DISCUSSION TITLE
Usage description - read the catalog entries from different EDC connectors

### DISCUSSION THREAD

**Discussion by oliverbusch** [eclipse-edc/FederatedCatalog#228] [Q&A] [answered]
What do you have to do to read the catalog entries from different EDC connectors?

**Comment by paullatzelsperger** [comment, marked as answer]
use the Query API:
```shell
curl --location '<HOST>:<PORT>/<BASEPATH>/v1alpha/catalog/query' \
--header 'Content-Type: application/json' \
--header 'X-Api-Key: <YOUR_API_KEY>' \
--data-raw '{
    "@context": {
        "edc": "https://w3id.org/edc/v0.0.1/ns/"
    },
    "@type": "QuerySpec"
}'
```

note that `<BASEPATH>` will be whatever you configured in `web.http.catalog.path`, <PORT> will be the value set in `web.http.catalog.port`

[edit]: that is assuming you want to collect the consolidated catalogs of several connectors that were crawled by your catalog instance.

**Comment by paullatzelsperger** [comment]
received no response, so I marked it as answer.
